ssl/ssl_ciph.c

```diff
@@ -1034,9 +1036,10 @@ static int ssl_cipher_process_rulestr(const char *rule_str,
             while (((ch >= 'A') && (ch <= 'Z')) ||
                    ((ch >= '0') && (ch <= '9')) ||
                    ((ch >= 'a') && (ch <= 'z')) ||
-                   (ch == '-') || (ch == '.') || (ch == '='))
+                   (ch == '-') || (ch == '.') || (ch == '=') || (ch == '_'))
```

Comment:
This is a comment on a commit from #1804 - so not relevant to this PR. Depending on where we end up with ciphersuite naming we might not need this change anyway...this was to allow "_" in ciphersuite names.